app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use App\Models\Category;
+use App\Models\Image;
+use Illuminate\Http\Request;
+use App\Http\Requests\PostRequest;
+use Illuminate\Support\Facades\DB;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $posts = Post::all();
+
+        return view('Admin/index', compact(['posts']));
```

Comment:
```suggestion
        return view('admin.index', compact(['posts']));
```
sửa tương tự chỗ khác